### DISCUSSION TITLE
Добавить перевод на Udemy

### DISCUSSION THREAD

**Discussion by Belachkin** [ilyhalight/voice-over-translation#299] [Ideas]
Было бы круто при просмотре курсов на Udemy что бы они переводились на русский язык, у меня много друзей разработчиков которые покупают там курсы и испытывают проблемы в понимании английского языка.

В общем и целом это было бы прекрасно <3

На всякий случай вот сайт: udemy.com

**Comment by ilyhalight** [comment]
В 1.4.1-beta5 уже есть поддержка udemy, единственное для перевода нужно устанавливать accessToken аккаунта, который нужен для получения ссылки на субтитры и видео с помощью api udemy

**Comment by Walterus** [comment]
Добавил по инструкции accessToken, страница перезагрузилась, но ничего не происходит,  снова нет данных для перевода и также запрашивается token!

**Reply by ilyhalight** [reply]
Создайте отдельный pr. и опишите проблему + Какую версию расширения вы используете
